puzzles/is-palindrome/isPalindrome.js

```diff
@@ -0,0 +1,26 @@
+// -----
+// Run time env (tested on):
+// Node v11.12.0
```

Comment:
We should also identify its node runtime versions, so that the user of this solution will be able to use the appropriate node version to run its solution